website/docs/policies/index.md

```diff
@@ -46,6 +46,21 @@ Starting with authentik 2022.11.0, the following checks can also be done with th
 -   Check the password hash against the database of [Have I Been Pwned](https://haveibeenpwned.com/). Only the first 5 characters of the hashed password are transmitted, the rest is compared in authentik
 -   Check the password against the password complexity checker [zxcvbn](https://github.com/dropbox/zxcvbn), which detects weak password on various metrics.
 
+### Password Uniqueness Policy
+
+Prevents users from reusing old passwords when changing their own password. The policy offers an option to limit the number many previous passwords to consider during evaluation.
```

Comment:
I think this reads awkwardly.  Suggestion: "This policy allows admins to specify how many previous password hashes should be kept to prevent re-use. By default, the password history depth is zero, permitting users to re-use any previous password."

Reply:
(Correct me if the default is wrong. I took that number from https://github.com/goauthentik/authentik/pull/10631/files#diff-cbb45e894de9c1957d2289a8301eaecdcef3ecd580a35d4fefba93f50441b398R30)

Reply:
I like that rephrasing a lot more and I'll get that changed!

The UI displays `1` as the default, which I can change to be `0` (I figured if you're setting up a the policy you want it to actually do something 😅 ). Is there a preference?

Reply:
Agree about defaulting to 1, from a usability perspective. Also zeros can be unnerving, lol.